src/containers/BooksForm.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Constants from '../common/constantModule';
+
+const BooksForm = () => (
+  <form className="bookform-section">
```

Comment:
- [ ] The BookForm should contain the handleSubmit function and should be attached to the form elemnt.